### PR TITLE
PLAT-129517: Added UI tests for VirtualList

### PR DESCRIPTION
**Pull request by daniel-stoian-lgp**
Enact-DCO-1.0-Signed-off-by: Daniel Stoian daniel.stoian@lgepartner.com

### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
 - Added ui tests for VirtualList

### Links
[//]: # (Related issues, references)
PLAT-129517

### Comments

### CODE REVIEW

File: tests/ui/specs/VirtualList/VirtualList/Wrap/VirtualListWrap-specs.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+const Page = require('../VirtualListPage');
+const {expectFocusedItem, waitUntilFocused} = require('../../VirtualList-utils');
+
+describe('Wrap', function () {
+	beforeEach(function () {
+		Page.open();
+	});
+
+	it('should wrap when', function () {
```
Comment (adrian-cocoara-lgp) [inline tests/ui/specs/VirtualList/VirtualList/Wrap/VirtualListWrap-specs.js:9]: Is this description incomplete?

File: tests/ui/specs/VirtualList/VirtualList/DisabledItem/VirtualListDisabledItem-specs.js [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+			Page.fiveWayToItem(index);
+			expect(Page.itemDisabled()).to.be.true();
+		}
+		Page.fiveWayToItem(index);
+		expect(Page.itemDisabled()).to.be.false();
+		// Spotlight displays on the previous Enabled item.
+		expectFocusedItem(0);
+	});
+
+	it('should navigate disabled and enabled items with native scrollmode [GT-29035]', function () {
```
Comment (adrian-cocoara-lgp) [inline tests/ui/specs/VirtualList/VirtualList/DisabledItem/VirtualListDisabledItem-specs.js:44]: Do we need the ticket name in description?

File: tests/ui/apps/VirtualList/VirtualList/VirtualList-View.js [hunk 3]
```diff
@@ -0,0 +1,223 @@
...
+	onChangeNumItems = ({value}) => {
+		this.setState({numItems: value});
+		updateDataSize(value);
+	};
+
+	onChangeSpacing = (obj) => {
+		this.setState({spacing: obj.value});
+	};
+
+	onChangeitemSize = ({value}) => {
```
Comment (adrian-cocoara-lgp) [inline tests/ui/apps/VirtualList/VirtualList/VirtualList-View.js:145]: Rename this `onChangeItemSize` and also where it is called

**Review by adrian-cocoara-lgp — APPROVED** [review]
LGTM. Great job! 👍

File: tests/ui/apps/VirtualList/VirtualGridList/VirtualGridList-View.js [hunk 4]
```diff
@@ -0,0 +1,201 @@
...
+
+class app extends React.Component {
+	constructor (props) {
+		super(props);
+		this.state = {
+			hideScrollbar: false,
+			horizontal: false,
+			noLabel: false,
+			numItems: 100,
+			minHeight: 450,
```
Comment (dongsuwon) [inline tests/ui/apps/VirtualList/VirtualGridList/VirtualGridList-View.js:76]: ImageItem's height is needed to adjust. 
Our samplers are made on a monstone basis, so we have to make them to fit the FHD (1920X1080) resolution. When minHeight 450 is applied to  the height value in "style={{height: ri.scaleToRem(minHeight * 3)}" becomes too large to display all imageItem on one screen and is truncated.

File: tests/ui/apps/VirtualList/VirtualGridList/VirtualGridList-View.js [hunk 5]
```diff
@@ -0,0 +1,201 @@
...
+			minWidth: 700,
```
Comment (dongsuwon) [inline tests/ui/apps/VirtualList/VirtualGridList/VirtualGridList-View.js:77]: same above

File: tests/ui/apps/VirtualList/VirtualGridList/VirtualGridList-View.js (hunk 4 above)
Comment (daniel-stoian-lgp) [reply tests/ui/apps/VirtualList/VirtualGridList/VirtualGridList-View.js]: Hello @dongsuwon , I updated the view with proper min/height for ImageItem

**Comment by dongsuwon** [comment]
jenkins passed:
enact-ui-tests/984/console

**Pull request merged**